### DISCUSSION TITLE
simple p-dropdown sample with data entered in html

### DISCUSSION THREAD

**Discussion by shougom-hennepin** [primefaces/community#819] [PrimeNG] [answered]
Im just looking for a super simple p-dropdown sample with the data entered in the html not the typescript array.

e.g  [options]='label:fieldname,value:pick me'

I for some reason CANNOT find any manual example of loading p-dropdown.  

If its not doable fine i just need to know.

**Comment by cetincakiroglu** [comment, marked as answer]
Hi,

Unfortunately, it's not possible. You can either pass an array of objects or an array of strings.
